src/analysis/ContextBar.ts

```diff
@@ -294,11 +346,17 @@ export const ContextBar = ({
                 color: !isTerminalVisible ? colors.dark(0.7) : contextBarStyles.contextBarButton.color,
               },
               hover: contextBarStyles.hover,
+              // @ts-expect-error
               'data-testid': 'terminal-button-id',
```

Comment:
Not sure why this is making typescript unhappy..

Reply:
Not sure either. I think this could also be related to the type parameter on Interactive's and Clickable's props types.